get-started/mcp/client-integrations.md

```diff
@@ -90,6 +120,12 @@ Before integrating with any client:
 <!-- Commenting this for safekeeping -->
 <!--See [Tools reference](/get-started/mcp/tools/){target=\_blank} for parameters and examples.-->
 
+## SDK integrations
+
+Looking to integrate MCP tools into your own applications? Check out our SDK integration guides:
```

Comment:
```suggestion
Looking to integrate MCP tools into your own applications? Check out the SDK integration guides:
```

Reply:
Avoid "we", "our", etc. in guides. Ok in tutorials, but should avoid everywhere else